splink/linker.py

```diff
@@ -2148,6 +2136,59 @@ def _compute_cluster_metrics(
 
         return df_cluster_metrics
```

Comment:
Don't think the name matters for practical purposes but suggest changing to `df_node_metrics` to help understanding and likewise on line 2135